doc/application/index.rst

```diff
@@ -299,19 +294,34 @@ The Zephyr build system compiles and links all components of an application
 into a single application image that can be run on simulated hardware or real
 hardware.
 
-As described in :ref:`getting_started_cmake`, on Linux and macOS you can choose
-between the ``make`` and ``ninja`` generators, whereas on Windows you need to use
-``ninja``. For simplicity we will use ``ninja`` throughout this guide.
+Like with any other CMake-based system, the build process takes place :ref:`in
+two stages <cmake-details>`. First build files (also known as buildsystem) are
```

Comment:
`First, build files (also known as a build system) are`, I think

Reply:
I used the CMake terminology here: https://cmake.org/cmake/help/latest/manual/cmake.1.html#introduction-to-cmake-buildsystems